README.md

```diff
@@ -100,8 +112,17 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
-```
+schema.sql exists in the db folder of a typical MVC layout which describes the has_many or belongs_to relationships of the classes placed in models.
```

Comment:
Want to caution that this answer is mainly in the context of rails, except we would have an `schema.rb` file in Rails. 
 Also, whenever I think of schema, I think of the structure of a relational database...

Basically a schema is essentially a blueprint for constructing a row entry in an SQL
database. 

Within the schema, one would assign the many (wishes) to the one (people)
by giving the wish-row a column for the person's ID, as the foreign_key connecting models.